modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt

```diff
@@ -506,6 +507,39 @@ class InstructionsTest : IrohaTest<Iroha2Client>() {
         assertEquals(50, result.assets[DEFAULT_ASSET_ID]?.value?.cast<AssetValue.Quantity>()?.u32)
     }
 
+    @Test
+    @WithIroha([AliceHas100XorAndPermissionToBurn::class])
+    @Feature("Assets")
+    @Story("Account burns an asset")
+    @Permission("no_permission_required")
+    @SdkTestId("burn_asset_for_account_in_same_domain")
+    fun `multi signature transaction`(): Unit = runBlocking {
+        val newBobKeyPair = generateKeyPair()
+        val newBobPublicKey = newBobKeyPair.public.toIrohaPublicKey()
+
+        client.tx(BOB_ACCOUNT_ID, BOB_KEYPAIR) {
+            mintSignatureCheckCondition(
+                BOB_ACCOUNT_ID,
+                SignatureCheckCondition.AllAccountSignaturesAnd(listOf(newBobPublicKey)),
+            )
+        }
+        client.tx(BOB_ACCOUNT_ID, BOB_KEYPAIR, newBobKeyPair) {
+            mintPublicKey(BOB_ACCOUNT_ID, newBobPublicKey)
+        }
+
+        val keyToSuccess = randomAlphabetic(5).asName()
+        val valueToSuccess = randomAlphabetic(5).asValue()
+        client.tx(BOB_ACCOUNT_ID, BOB_KEYPAIR, newBobKeyPair) {
```

Comment:
that's probably the most obvious case, what if send 2 transactions with 1 signatures each or smth like that? we should check with Iroha team

Reply:
from Roman: "user send transaction with the exact same payload but different signature, multisig transactions are aggregated on iroha side until all required signatures are gathered."

so let's refactor the test to 2 different transactions sent with a delay between each other

Reply:
Marin: "yes, signatures can be attached over time, but you have to use get_original_transaction or pending_transactions to get the original transaction then add signature to it" 

so even better to use `pending_transaction` to get the transaction from Iroha to sign that with the second key